packages/freezed/README.md

```diff
@@ -44,7 +44,7 @@ to focus on the definition of your model.
       - [Extending classes](#extending-classes)
       - [Defining a mutable class instead of an immutable one](#defining-a-mutable-class-instead-of-an-immutable-one)
       - [Allowing the mutation of Lists/Maps/Sets](#allowing-the-mutation-of-listsmapssets)
-    - [Classic classes](#classic-classes)
+    - [Classic classes (NEW)](#classic-classes-new)
```

Comment:
I don't like that (new) stuff.

If folks want to know what's new, there's the changelog